src/editor/components/monaco-tools/__tests__/code-chunk-identifiers.test.js

```diff
@@ -0,0 +1,114 @@
+import { fetchIdentifiers, cssIdentifiers } from "../code-chunk-identifiers";
```

Comment:
I wonder if it might make more sense to just have an `iomd-editor` directory to just put all this stuff in:

`src/editor/components/iomd-editor/` 

Then could just dump all these extra js files in there, along with `iomd-editor.jsx`. I don't think it would be necessary to say these are "monaco-tools" in the directory structure then.

Reply:
yep good idea